0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -33,31 +33,37 @@ Any SPSP receiver will run an SPSP server and expose an HTTPS endpoint called th
 3. The sender constructs an ILP payment using the receiver's ILP address.
-4. The sender uses PSK to generate the payment condition and format additional data intended for the reciever to be sent with the payment.
-5. The sender prepares a local transfer to a connector with the condition and ILP packet attached.
-6. The receiver's PSK module registers the incoming transfer, parses and validates the ILP packet and the incoming transfer.
-7. The receiver MAY submit the incoming payment to an external system for review to ensure that the funds are wanted.
-8. If the payment is expected, the receiver's PSK module submits the condition fulfillment to the ledger to execute the transfer. If not, the PSK module rejects the incoming transfer.
-9. If the receiver executed the transfer, the sender's SPSP client receives a notification from its ILP module that the transfer has been executed, including the condition fulfillment from the receiver, and notifies the sender that the payment is completed. If the receiver rejected the transfer, the sender's SPSP client receives a notification with the receiver-specified error message detailing why the payment was rejected.
+4. The sender uses PSK2 to generate the payment chunk and format additional data intended for the reciever to be sent with the payment.
+5. The sender sends a prepare packet to a connector with the condition and ILP address attached.
+6. The receiver's PSK2 module registers the incoming packet, parses, and validates the ILP packet.
+7. The receiver MAY submit the incoming packet to an external system for review to ensure that the funds are wanted.
+8. If the payment is expected, the receiver's PSK2 module replies with a fulfillment packet. If not, the PSK2 module replies with a reject packet.
+9. The receiver MAY update a balance related to this SPSP receiver, i.e. in the case of an invoice.
+10. The fulfillment or rejection packet comes back to the sender.
+11. The sender MAY send further payment chunks with PSK2 (repeating steps 4-10).
 
 ## Specification
 
-The SPSP endpoint is a URI used by the sender to query information about the receiver and set up payments. The SPSP endpoint URI MAY contain query string parameters. The sender SHOULD treat the URI as opaque.
+The SPSP endpoint is a URI used by the sender to query information about the receiver and set up payments. The SPSP endpoint URI MAY NOT contain query string parameters. The sender SHOULD treat the URI as opaque. There are several supported ways to refer to an SPSP endpoint:
+
+- [Payment-pointer](#appendix-b-payment-pointer) (Recommended) `$alice.example.com` or `$example.com/bob`. This SHOULD be the only kind of SPSP identifier exposed to users.
+- [Webfinger acct](#appendix-a-deprecated-webfinger-discovery) (Deprecated) `alice@example.com`.
```

Comment:
We shouldn't have two types of identifiers. if we're adding the payment pointers, let's get rid of the webfinger identifiers.